packages/apollo-voyager-conflicts/src/api/ObjectConflictError.ts

```diff
@@ -19,6 +19,11 @@ export interface ConflictData {
    */
   clientData?: ObjectStateData
 
+  /**
+   * Base data that before server side changes that caused conflict were applied
+   */
+  baseData?: ObjectStateData
```

Comment:
Hey @wtrocki sorry if this is a silly question, but what exactly is the `baseData`? I'm not 100% sure.

Could you maybe describe an example where the base data is returned in the error and how it would be different from the server data and client data?

Reply:
When resolving conflicts we usually have 3 types of artifacts:
- Input data (usually comes from client)
- Current source of truth data
- Base for both changes

There are numerous examples of this in real world. 
For example any type of merge client we tend to see 3 artifacts:
https://chodounsky.net/resources/img/posts/2013-05-09-merge-files-with-git-merge-tool-1.png